src/util/tile_request_cache.js

```diff
@@ -151,6 +151,7 @@ export function enforceCacheSizeLimit(limit: number) {
 
     sharedCache
         .then(cache => {
+            if (!cache) return;
```

Comment:
Is this related to the Windows installation fix? If not, it should probably be in its own Pull Request :)